src/gui/org/deidentifier/arx/gui/view/impl/define/ViewAttributeList.java

```diff
@@ -458,9 +492,11 @@ private void updateEntries() {
             TableItem item = new TableItem(table, SWT.NONE);
             item.setText(new String[] { "", attribute, getDataType(attribute), getDataTypeFormat(attribute) }); //$NON-NLS-1$
             AttributeType type = model.getInputDefinition().getAttributeType(attribute);
-            item.setImage(0, controller.getResources().getImage(type));  
-            if (model.getSelectedAttribute() != null && model.getSelectedAttribute().equals(attribute)) {
```

Comment:
Why has the part where the selected attribute is also selected in the table been removed? Please implement a specific ModelEvent as described above, then you don't need to do crazy hacks with selected attributes, which means that you can probably keep this. Please change as little as possible.